zig/private/common/zig_build.bzl

```diff
@@ -224,6 +224,9 @@ def zig_build_impl(ctx, *, kind):
             # For now, linking with cc_common.link implies linking with libc.
             # But this should probably be made configurable.
             "-lc",
+            # This is implied too unless disabled further down.
+            # This is because zig compiler-rt ships with __zig_probe_stack which doesn't exist in regular compiler-rt.
+            "-fcompiler-rt",
```

Comment:
I've been wondering whether `-fcompiler-rt` would be better controlled as a build setting rather than an attribute. This looks like an indicator that a build setting would be better. Then users could `zig_configure_*` it together with `use_cc_common`.

Reply:
Again, my main issue is that non toolchain developers will have a very hard time understanding where the issue comes from if this was not implicit. 

This is the error they would get with no further instructions:

```
Undefined symbols for architecture x86_64:
  "___zig_probe_stack", referenced from:
      _std.debug.printLineInfo in x.o)
      _std.sort.sort in x.o)
ld: symbol(s) not found for architecture x86_64
```

Reply:
Right, let's stick with this for now as well.